ci-operator/step-registry/upi/conf/gcp/upi-conf-gcp-commands.sh

```diff
@@ -65,6 +88,34 @@ data = yaml.load(open(path));
 data["spec"]["mastersSchedulable"] = False;
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 
+### Remove DNS Zones (optional)
+if [[ -v IS_XPN ]]; then
+  echo "Removing DNS Zones..."
+  python -c '
+import yaml;
+path = "manifests/cluster-dns-02-config.yml";
+data = yaml.load(open(path));
+del data["spec"]["privateZone"];
+open(path, "w").write(yaml.dump(data, default_flow_style=False))'
+fi
+
+### Update the cloud-provider manifest ([Shared VPC (XPN)][sharedvpc] only)
+if [[ -v IS_XPN ]]; then
+  echo "Updating the cloud-provider manifest..."
+  sed -i "s/    subnetwork-name.*/    network-project-id = ${HOST_PROJECT}\\n    network-name    = ${HOST_PROJECT_NETWORK_NAME}\\n    subnetwork-name = ${HOST_PROJECT_COMPUTE_SUBNET_NAME}/" manifests/cloud-provider-config.yaml
+fi
+
+### Enable external ingress (optional)
```

Comment:
I think `(optional)` does not belong here. Leftover copy/paste perhaps.

Reply:
These comment strings line up with the repo documentation, making it easier to compare the two files using `diff` for audit purposes. Keeping the same is intentional.

https://github.com/openshift/installer/blob/master/docs/user/gcp/install_upi.md

Reply:
> Keeping the same is intentional.

personally i feel like this conditional is just a little too distracting, but let's table the need to split as follow up if we do decide to do it.